src/ui/Analytics/AnalyticsActionListMeta.ts

```diff
@@ -1393,5 +1393,21 @@ export var analyticsActionCauseList = {
   removeMissingTerm: <IAnalyticsActionCause>{
     name: 'removeMissingTerm',
     type: 'missingTerm'
+  },
+  /**
+   * Identifies the search event that gets logged when a user clicks a query suggestion based on the usage analytics recorded queries.
+   *
+   * `actionCause`: `'QuerrySuggestPreviewQuery'`
+   * `actionType`: `'querySuggestPreview'`
+   *
+   * Logging an event with this actionType also adds the following key-value pairs in the custom data property of the Usage Analytics HTTP service request.
+   * `"partialQuery":`: <correspondingPartialQuery>
+   * `"suggestionRanking":`: <suggestionRankingValue>
+   * `"partialQueries":`: <correspondingPartialQueries>
+   * `"suggestions":`: <availableQuerySuggestions>
+   */
+  QuerrySuggestPreviewQuery: <IAnalyticsActionCause>{
```

Comment:
Query is a bit repetitive I find. You could maybe remove the 2nd one. 
The first Query has a typo.

Reply:
🤔 `QuerySuggestPreview` is the name of the component and doesn't mean anything.. 
I think I'll go with `ShowQuerySuggestPreview`